src/pages/WalletProviders.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  Card,
+  Heading,
+  Icon,
+  Toggle,
+  Title,
+  Notification,
+  Modal,
+  Button,
+} from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+import { InfoTooltip } from "components/InfoTooltip";
+import { SectionHeader } from "components/SectionHeader";
+import { useRedux } from "hooks/useRedux";
+import { useState, useEffect } from "react";
+import { AppDispatch } from "store";
+import { getWalletsAction, updateWalletAction } from "store/ducks/wallets";
+
+export const WalletProviders = () => {
+  const [updateWalletModal, setUpdateWalletModal] = useState({
+    visible: false,
+    walletId: "",
+    enabled: false,
```

Comment:
We could make these a param (`initState`, for example) to use here and when we need to reset.